tox.ini

```diff
@@ -3,13 +3,15 @@ envlist = py35, py36, flake8
 
 [travis]
 python =
-    3.6: py36
+    3.6: py36, flake8
     3.5: py35
 
+; This does not really work, moved flake8 check to .travis.yml
 [testenv:flake8]
-basepython = python
+basepython = python3.6
 deps = flake8
-commands = flake8 src
+commands = flake8 mlbench_core
```

Comment:
There's https://github.com/tholo/pytest-flake8#configuring-flake8-options-per-project-and-file so we could run it as part of pytest with tox instead, there's also https://github.com/shopkeep/pytest-black if we switch to `black`